documentation/en/api-v0-methods-worker.md

```diff
@@ -167,6 +167,7 @@ Inputs:
   },
   null,
   1024,
+  null,
```

Comment:
This shouldn't change. Do we need to stub out a version of the v0 API that passes nil?

Reply:
Yeah... but is there any real reason to? Alternatively, if it's internal, is there any way to make sure it's always in the "unstable" API, and never gets stabilized?

Reply:
(doesn't really block merge)